src/System.Management.Automation/CoreCLR/CorePsAssemblyLoadContext.cs

```diff
@@ -483,6 +531,41 @@ private void ThrowFileNotFoundException(string errorTemplate, params object[] ar
             throw new FileNotFoundException(message);
         }
 
+        private static string s_nativeDllSubFolder;
+        private static string s_nativeDllExtension;
+
+        private static string GetNativeDllSubFolderName(out string ext)
+        {
+            string folderName = string.Empty;
+            ext = string.Empty;
+            var processArch = RuntimeInformation.ProcessArchitecture switch
```

Comment:
Can you just use `RuntimeInformation.ProcessArchitecture.ToString().ToLower()`?

Reply:
Done.